docs/getting-started/continuous_integration.md

```diff
@@ -62,7 +62,7 @@ jobs:
      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
      with:
        oss-fuzz-project-name: 'example'
-       fuzz-time: 600
+       fuzz-seconds: 600
        dry-run: false
```

Comment:
What happens if someone leaves dry-run out? will things still work?

Reply:
The default is false if it is left out.